Userland/Libraries/LibWeb/Painting/Paintable.h

```diff
@@ -212,6 +217,7 @@ class Paintable
     Optional<JS::GCPtr<Layout::Box const>> mutable m_containing_block;
 
     OwnPtr<StackingContext> m_stacking_context;
+    Optional<u32> m_alpha_mask_id;
```

Comment:
shouldn't alpha mask id be only stored in TextPaintable?

Reply:
Both `PaintableBox` and `TextPaintable` need the alpha mask id, since it maps to both the alpha mask and the background canvas that `PaintableBox` needs to paint the background layer to. So I can move this to just those two classes instead of the parent if thats preferred?

Also the variable name might be a bit 💩, just haven't come up with one that clearly explains what the id maps to.